cf-custom-resources/lib/dns-cert-validator.js

```diff
@@ -78,43 +82,49 @@ let report = function (
   });
 };
 
+const aliasesChanged = async function (
+    oldResourceProperties,
+    aliases,
+) {
+  if (!oldResourceProperties) {
+    return true;
+  }
+  let prevAliases = await getAllAliases(
+      oldResourceProperties.Aliases
+  );
+  let aliasesToDelete = [...prevAliases,].filter(function (itm) {
+    return !aliases.has(itm);
+  });
+  let aliasesToAdd = [...aliases,].filter(function (itm) {
+    return !prevAliases.has(itm);
+  });
+  return !aliasesToAdd.length + aliasesToDelete.length === 0;
```

Comment:
😮  is this doing the right thing vs:
```suggestion
  return !((aliasesToAdd.length + aliasesToDelete.length) === 0);
```

Reply:
Maybe just
```
return aliasesToAdd.length + aliasesToDelete.length !== 0;
```